src/ExpressionEvaluator/Package/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="21BAC26D-2935-4D0D-A282-AD647E2592B5" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Expression Evaluators</DisplayName>
     <Description xml:space="preserve">Roslyn Expression Evaluators</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\Expression Evaluators</ShortcutPath>
```

Comment:
Good question: `<ShortcutPath>` is relative to the `Common7\IDE\Extensions` directory.  This value was based on the value chosen by another internal MS team that's already done this work (see me offline for details.)  The short version is that `Extensions` is reserved for random VSIX packages that could be installed/uninstalled at any given moment and `CommonExtensions` is to be used for more permanent packages.